PYTHON/AzureSQLDB/read_from_azure_sql_db.py

```diff
@@ -0,0 +1,30 @@
+# This sample allows you to read data from Azure SQL database 
+# Complete these steps before testing this funtion 
+#   1. Select Manage connections to connect to Azure SQL database 
+#   2. Copy the Alias name and replace it below
+
+@app.connection("sqlDB",alias="<alias for azure sql database>")
```

Comment:
This attribute (in this sample and in all others) is currently called `app.fabric_item_input(...)`, but might soon be changing to `connection`

Reply:
@marshall-lerner  yes , for now I shall keep this and will change once your changes are in